src/sugar3/graphics/toolcombobox.py

```diff
@@ -18,15 +18,32 @@
 """
 STABLE.
 """
-
 from gi.repository import Gtk
 from gi.repository import GObject
 
 from sugar3.graphics.combobox import ComboBox
 from sugar3.graphics import style
+'''
+A ToolComboBox is a widget that allows the user to
+choose from a list of valid choices.
```

Comment:
Too brief.  What makes a ToolComboBox different to a ComboBox?  You might like to compare against the documentation in combobox.py so that your new documentation is consistent.